package.json

```diff
@@ -27,13 +27,17 @@
   "license": "MIT",
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
+    "browserify": "^13.0.0",
```

Comment:
`dependencies` must now specify fixed versions. For `devDependencies`, I forgot what @xPaw and I concluded on though.